sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -290,15 +308,21 @@ export class EventHubClient {
   }
 
   /**
-   * Creates an EventHubConsumer that can be used to receive events from the Event Hub for which this
-   * client was created.
+   * Creates an Event Hub consumer responsible for reading `EventData` from a specific Event Hub partition,
+   * in the context of a specific consumer group.
+   *
+   * Multiple consumers are allowed on the same partition in a consumer group.
+   * If there is a need to have an exclusive consumer for a partition in a consumer group,
+   * then specify the `ownerLevel` in the `options`.
+   * Exclusive consumers were previously referred to as "Epoch Receivers".
+   *
+   * Designating a consumer as exclusive may be specified in the `options` via `ownerLevel`.
+   * By default, consumers are created as non-exclusive.
```

Comment:
> Designating a consumer as exclusive may be specified in the `options` via `ownerLevel`.
By default, consumers are created as non-exclusive.

Do we need this anymore after the changes to the previous section that cover the same?

Reply:
Removed since this is implied by the line above it.